### PR TITLE
Adding some suggestions

### PR DESCRIPTION
**Pull request by ausias-armesto**
- Rename scripts to not start by capital letters
- Adding error handling at the start of every script
- Setting variable names for working within the functions instead of variable numbers

### CODE REVIEW

File: setup-qubes.sh [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+	fi
+	VMNAME="${1}"
+	FILE="${2}"
+	EXE="${3}"
+
+	echo "Fetching ${FILE} from VM ${VMNAME}..."
+	
+	# delete file in case it already exists on dom0 and ignore errors
+	FILENAME=$(basename "$FILE")
+	rm $FILENAME 2>>/dev/null
```
Comment (ausias-armesto) [inline setup-qubes.sh:23]: Removing $FILENAME might fail, as the filepath for the file is not found when the $FILE contains subdirectories

File: setup-qubes.sh [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+	# fetch the file via the 'cat' hack to avoid dom0 security precautions 
+	if qvm-run -p ${VMNAME} cat ${FILE} >> $FILENAME; then
```
Comment (ausias-armesto) [inline setup-qubes.sh:26]: The `qvm-run` command needs to be installed. It's a good practice to create some kind of check function executed at the beginning of the script where to check if special command lines are installed or not.

File: setup-qubes.sh [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+	qvm-start ${APP_VM}
+	
+	echo "trying to fetch ${APPNAME} appVM install files...."
+	fetchRunClean ${APP_VM} ${APPNAME} /home/user/SEQS/install-scripts/ ${APPNAME}_appVM.sh
+
+	echo "shutting app VM down..."
+	qvm-shutdown ${APP_VM}
+}
+
+cd ~
```
Comment (ausias-armesto) [inline setup-qubes.sh:113]: It is not a good practice to force scripts to run in home directory. I suppose that this is still draft version of the scripts.

File: setup-qubes.sh [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+installApp brave red
+installApp element red
+installApp keepass black offline
+installApp docker red
+installApp signal red
+installApp telegram red
+installApp wallets orange
+
+# finally delete this setup file after running it
+rm $0
```
Comment (ausias-armesto) [inline setup-qubes.sh:124]: It is not documented this parameter and not known their usage in the whole script.

File: setup-qubes.sh [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+		echo "taking app VM offline..."
+		sleep 2
+		qvm-prefs ${APP_VM} netvm none
+	fi
+
+	echo "trying to fetch ${APPNAME}.desktop file..."
+	if fetchFromVm personal /home/user/SEQS/menu-files/${APPNAME}.desktop; then
+		echo "moving ${APPNAME}.desktop file to template VM..."
+		qvm-move-to-vm ${TEMPLATE_VM} ${APPNAME}.desktop
+		qvm-run -p ${TEMPLATE_VM} sudo mv /home/user/QubesIncoming/dom0/${APPNAME}.desktop /usr/share/applications/
```
Comment (ausias-armesto) [inline setup-qubes.sh:94]: The path `/home/user/QubesIncoming/dom0` will be in the target environment when executing this script, as this won't work for mac users for instance.

Comment (SCBuergel) [reply setup-qubes.sh:94]: indeed, this is all super opinionated to only work on QubesOS

File: setup-qubes.sh (hunk 2 above)
Comment (SCBuergel) [reply setup-qubes.sh:26]: good point - will add this. This should be a given on QubesOS but maybe user is running this not in dom0 and then it should not just do undefined things.

**Review by SCBuergel — APPROVED** [review]

**Pull request merged**